packages/hooks/feed/useSocialReactions.ts

```diff
@@ -75,6 +81,8 @@ export const useSocialReactions = ({
       func: "ReactPost",
       args: [TERITORI_FEED_ID, post.id.split("-")[1], emoji, "true"],
     };
+
+    setLoading(true);
     const txHash = await adenaDoContract(
```

Comment:
This can throw, the waitForTransaction also probably

Reply:
Yes https://github.com/TERITORI/teritori-dapp/pull/1393/commits/269f23fa12b9016eb304b9c97d64bb8a322807ff